docs/integrations/azure/discovery.md

```diff
@@ -0,0 +1,49 @@
+---
+id: discovery
```

Comment:
To make sure that this actually gets visible on the microsite, you want to add the proper references in `microsite/sidebars.json` and `/mkdocs.yml`.